torch/csrc/jit/python/pybind_utils.cpp

```diff
@@ -114,6 +114,16 @@ IValue toIValue(py::handle obj, const TypePtr& type, c10::optional<int32_t> N) {
       if (torch::is_symfloat(py::handle(obj))) {
         return py::cast<c10::SymFloat>(obj).guard_float(__FILE__, __LINE__);
       }
+      if (THPVariable_Check(obj.ptr())) {
+        auto var = py::cast<autograd::Variable>(obj);
```

Comment:
What ended up necessitating these changes? (just trying to understand how these relate to the pending_fresh_unbacked_symbols bits)